Classes/Command/ProjectionCommandController.php

```diff
@@ -49,54 +54,90 @@ public function listCommand()
+            $this->outputLine();
+            $this->outputLine('<b>PROJECTOR:</b>');
+            $this->outputLine('  %s', [$projection->getProjectorClassName()]);
+            $this->outputLine();
 
-        $this->outputLine('<b>HANDLED EVENT TYPES:</b>');
-        foreach ($projection->getEventTypes() as $eventType) {
-            $this->outputLine('  * %s', [$eventType]);
+            $this->outputLine('<b>HANDLED EVENT TYPES:</b>');
+            foreach ($projection->getEventTypes() as $eventType) {
+                $this->outputLine('  * %s', [$eventType]);
+            }
+        } catch (\Exception $e) {
+            $this->outputLine('<error>%s</error>', [$e->getMessage()]);
+            $this->quit(1);
         }
     }
 
     /**
-     * @param string $text
-     * @param int $maximumCharacters
-     * @return mixed
+     * Replay projections
+     *
+     * This command allows you to replay all relevant events for one or all projections.
+     * You can specify the special identifier "all" for replaying _all_ existing projections.
+     *
+     * @param string $projection The projection identifier; see projection:list for possible options
+     * @return void
+     * @see neos.cqrs:projection:list
      */
-    private function truncate($text, $maximumCharacters = 36)
+    public function replayCommand($projection)
     {
-        $length = strlen($text);
-        if ($length <= $maximumCharacters) {
-            return $text;
+        $eventsCount = 0;
+
+        try {
+            if ($projection === 'all') {
+                foreach ($this->projectionManager->getProjections() as $projection) {
+                    $this->outputLine('Replaying events for %s ...', [$projection->getFullIdentifier()]);
+                    $eventsCount += $this->projectionManager->replay($projection->getFullIdentifier());
+                }
+            } else {
+                $projection = $this->projectionManager->getProjection($projection);
+                $this->outputLine('Replaying events for %s ...', [$projection->getFullIdentifier()]);
+                $eventsCount = $this->projectionManager->replay($projection);
+            }
+            $this->outputLine('Replayed %s events.', [$eventsCount]);
+        } catch (\Exception $e) {
+            $this->outputLine('<error>%s</error>', [$e->getMessage()]);
+            $this->quit(1);
         }
-        return substr_replace($text, '...', ($maximumCharacters - 3) / 2, $length - $maximumCharacters + 3);
     }
 
     /**
-     * @param string $identifier
-     * @return void
+     * Shortens the given text by removing characters from the middle
+     *
+     * @param string $text Text to shorten
+     * @param int $maximumCharacters Maximum of characters
+     * @return string
      */
-    public function replayCommand($identifier)
+    private function shortenText($text, $maximumCharacters = 36)
```

Comment:
funny, that "private" wasn't on purpose, don't know how it slipped in. But well, doesn't matter anyway.